test/writeFunctions.test.ts

```
@@ -394,6 +394,22 @@ describe("Write Functions Client Tests", () => {
         hatId: hatX_1_2,
         newEligibility: agreementInstance2,
       });
+
+      // set hats as claimable in MCH for signAgreementAndClaim tests
+      // ClaimType.ClaimableFor = 2
+      const mchModule = hatsModulesClient.getModuleById(MCH_MODULE_ID) as Module;
+      const setHatsClaimabilityFunc = mchModule.writeFunctions.find(
+        (f) => f.functionName === "setHatsClaimability",
+      );
```

ESLint

[error] Replace `⏎········(f)·=>·f.functionName·===·"setHatsClaimability",⏎······` with `(f)·=>·f.functionName·===·"setHatsClaimability"`

(prettier/prettier)